osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerMatchSettingsOverlay.cs

```diff
@@ -222,12 +246,30 @@ private void load(OverlayColourProvider colourProvider, OsuColour colours)
                                                         }
                                                     },
                                                 },
-                                                initialBeatmapControl = new BeatmapSelectionControl
+                                                playlistContainer = new FillFlowContainer
```

Comment:
I guess this is a simplification of code? Is the plan to make this further editable on this screen though?

Weird side effect which is funny (doesn't require fixing):

https://user-images.githubusercontent.com/191335/142571413-df71a675-d4e8-453d-95dc-f8bb454f0070.mp4

Reply:
The logic just doesn't work well otherwise (using `BeatmapSelectionControl`), it was much too complicated which seemed super unnecessary.